File: Java/Sum_Of_Digits_Of_A_Number.java
// Coded by: rohan09-raj (GitHub Username)
import java.util.Scanner;

public class SumOfDigits {

  public static void main(String args[]) {
    int number, digit, sum = 0;
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter the number: ");
    number = sc.nextInt();
    while (number > 0) {
      digit = number % 10;
      sum = sum + digit;
      number = number / 10;
    }

    System.out.println("Sum of Digits: " + sum);
  }
}

File: sum of digits of a number.cpp
#include <iostream>
using namespace std;

int main()
{
     int num;

     cout << "Enter a positive number: ";
     cin >> num;



     int sum=0,m;    
  
	while(num>0)    
	{    
		m=num%10;    
		sum=sum+m;    
		num=num/10;    
	}    
	cout<<"Sum is= "<<sum<<endl;    
	return 0;  
}  
